### PR TITLE
added automatically log delete & module will update time interval

### PR DESCRIPTION
**Pull request by rfnshare**
## PR Type

Feature

## PR Checklist

- [x] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] A changelog entry has been made.
- [ ] Version number has been updated.
- [x] Required modules have been added to respective "requirements*.txt" files.
- [ ] Relevant Test Cases added to this description (below).
- [ ] (Team) Label with affected action categories and server status.

## Overview

- Added auto log delete based one time you set
- Added module update time interval. User can set the frequent time for module update.

No

### CODE REVIEW

File: Framework/settings.conf [hunk 1]
```diff
@@ -5,22 +5,25 @@ password =
...
 available_to_all_project = False
 _file = temp_config.ini
 _file_upload_path = TestExecutionLog
+last_module_update_date =
+last_log_delete_date =
 
 [Selenium_driver_paths]
-chrome_path = 
-firefox_path = 
-edge_path =
```
Comment (Muntasib-creator) [inline Framework/settings.conf:18]: revert as before

File: Framework/settings.conf [hunk 2]
```diff
@@ -5,22 +5,25 @@ password =
...
+chrome_path =
+firefox_path =
+edge_path =
+opera_path =
+ie_path =
+electron_chrome_path =
 
 [Inspector]
-Window = 
+Window = ""
```
Comment (Muntasib-creator) [inline Framework/settings.conf:27]: revert as before

**Comment by sazid** [comment]
@rfnshare you need to describe the feature and put appropriate title.

File: requirements.txt [hunk 3]
```diff
@@ -47,4 +47,5 @@ google-cloud-bigquery
 google-cloud-bigquery-storage
 xvfbwrapper
 deepdiff
-xmltodict
\ No newline at end of file
+xmltodict
+configobj
```
Comment (sazid) [inline requirements.txt]: We don't need additional configobj to accomplish this. There is already a builtin config handler module inside node for these use cases, please use that.

**Review by sazid — CHANGES_REQUESTED** [review]

**Review by sazid — APPROVED** [review]

**Pull request merged**